scripts/install_elk.sh

```diff
@@ -0,0 +1,88 @@
+sudo add-apt-repository -y ppa:webupd8team/java
+sudo apt-get update
+sudo apt-get -y install oracle-java8-installer
+wget -O - http://packages.elasticsearch.org/GPG-KEY-elasticsearch | sudo apt-key add -
+echo 'deb http://packages.elasticsearch.org/elasticsearch/1.4/debian stable main' | sudo tee /etc/apt/sources.list.d/elasticsearch.list
+sudo apt-get update
+sudo apt-get -y install elasticsearch=1.4.4
```

Comment:
Can you remove all the sudo's?  It looks like there are a mixture of commands where some assume root access and some use sudo.  I think the entire script should just be run as root to keep it simple.